modules/bullet/godot_ray_world_algorithm.cpp

```diff
@@ -102,7 +100,7 @@ void GodotRayWorldAlgorithm::processCollision(const btCollisionObjectWrapper *bo
 
 		btScalar depth(ray_shape->getScaledLength() * (btResult.m_closestHitFraction - 1));
 
-		if (depth >= -RAY_STABILITY_MARGIN)
+		if (depth >= -ray_shape->getMargin())
 			depth = 0;
```

Comment:
can you please change this with: `depth *= 0.5;`

Reply:
What would be the rationale for that?